packages/sp1-ics07-tendermint-utils/Cargo.toml

```diff
@@ -22,3 +22,6 @@ tendermint-rpc                   = { workspace = true }
 tendermint                       = { workspace = true }
 alloy-signer-local               = { workspace = true }
 alloy-network                    = { workspace = true }
+# We need to use 0.11 until tendermint-rpc is updated to use 0.12
+reqwest            = { version = "0.11", default-features = true }
```

Comment:
You can still import multiple versions of reqwest in the top level `Cargo.toml` and call it something else, like `reqwest_v0_11`. I know what you did here is also commonly done. But I think we should continue with keeping everything in top level `Cargo.toml`

Reply:
I was not aware that you could do that. Will try that!